### DISCUSSION TITLE
Request for react-dropzone update due to custom MIME type filtering issue on Windows

### DISCUSSION THREAD

**Discussion by rapolaslisboa** [mantinedev/mantine#6975] [Releases]
Hey, thank you for the great library! I’ve encountered an issue with the `Dropzone` component that appears to be specific to Windows (tested on macOS and works fine):

When some custom MIME types are added to the accept property of the `Dropzone` component for certain file types (in my case, font files such as .font), the file picker on Windows does not restrict the displayed file types to just fonts. Instead, it shows all types of files, even when only font MIME types are specified. This behavior is not expected, as the picker should only display the allowed font files.

Additionally, the `Dropzone` component has a rejection status, which does not show up on Windows when attempting to drag and drop a file that is not of the font type, in our example (check the font upload feature in the Branding application). This further complicates the user experience, as there is no visual feedback for unsupported file types.

This issue seems to occur only on Windows and appears to be related to the `react-dropzone` library, as suggested by several discussions on GitHub:

- [Mantine Issue #2814](https://github.com/mantinedev/mantine/issues/2814)
- [Mantine Issue #2864](https://github.com/mantinedev/mantine/issues/2864)
- [Mantine Issue #5443](https://github.com/mantinedev/mantine/issues/5443)
- [Mantine Issue #6608](https://github.com/mantinedev/mantine/issues/6608)
- [Mantine Issue #1809](https://github.com/mantinedev/mantine/issues/1809)

I understand that this might not be directly fixable on Mantine’s side. However, I noticed that `react-dropzone` has recently resumed updates after two years of inactivity. I was wondering if there are any plans to update the `react-dropzone` package in an upcoming Mantine release (I also noticed you're currently using a forked version, `react-dropzone-esm`). It would be great to benefit from the new or upcoming `react-dropzone` releases, as they might include relevant fixes.

Thank you!

**Comment by rtivital** [comment]
react-dropzone in its current state cannot be used with modern frameworks, see https://github.com/react-dropzone/react-dropzone/issues/1259

I also do not see any of the above issues addressed in the recent releases of react-dropzone – https://github.com/react-dropzone/react-dropzone/releases

If there is value/actual bug fixes and react-dropzone properly supports esm, then `@mantine/dropzone` can be migrated back to react-dropzone.
